tests/new_signature_spec/signature_input_text_multiple.json

```diff
@@ -0,0 +1,113 @@
+{
+  "description": "Tests the signature specification at https://kite.quip.com/NdIbAwg272aJ/Spec-Function-Signatures-Editor-Plugins-UIUX-WIP, inserting using multiple characters at once",
+  "setup": {
+    "whitelist": [
+      "data/whitelisted"
+    ],
+    "kited": "authenticated",
+    "routes": [
+      {
+        "match": "^/clientapi/editor/signatures$",
+        "response": {
+          "status": 200,
+          "body": "data/responses/signature.json"
+        }
+      },
+      {
+        "match": "^/clientapi/plan$",
+        "response": {
+          "status": 200,
+          "body": "data/responses/plan.json"
+        }
+      }
+    ]
+  },
+  "test": [
+    {
+      "description": "opening a file",
+      "step": "action",
+      "type": "new_file",
+      "properties": {
+        "file": "data/whitelisted/file.py",
+        "content": "foo("
+      }
+    },
+    {
+      "description": "moving cursor at end of identifier",
+      "step": "action",
+      "type": "move_cursor",
+      "properties": {
+        "offset": 4
+      }
+    },
+    {
+      "description": "typing 'bar'",
```

Comment:
This won't work in VSCode, as I explained yesterday, VSCode will only calls signatures providers when typing a `(` or a `,` (as we specified in the provider registration), when the panel is visible any input will trigger a new request, but given the context (the file was open containing an already existing function call) the panel won't be shown and all following inputs won't trigger signature requests.